src/style.css

```diff
@@ -1 +1,90 @@
 /* Styles go here */
+html {
+    --active-color: #00acdc;
+    --font-color: black;
+    --bg-color: #fff;
+}
+
+body {
+    margin: 0;
+}
+
+header {
+    display: flex;
+    padding: 0 50px;
+    font-family: Roboto,sans-serif;
+    align-items: center;
+    justify-content: space-between;
+    background: var(--bg-color);
+    box-shadow: 0 2px 4px rgb(0 0 0 / 5%);
+}
+
+.logo {
+    display: block;
+    width: 40px;
+    height: 40px;
+}
+
+.nav {
+    display: flex;
+    justify-content: center;
+    justify-items: center;
+}
+
+.nav__list {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+.nav__item {
+    margin-left: 20px;
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: 60px;
+    font-family: Roboto,sans-serif;
```

Comment:
can be removed

Reply:
the same issue, just font family can be removed